CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.6.0] - 2023-04-06
+### Added
+  * Added support for ID-mapped overlayfs lower layers; eliminates need for shiftfs and Sysbox rootfs chown; requires kernel 5.19+.
+  * Have Sysbox perform shiftfs and ID-mapping functional checks during init (issue #596).
+  * Fix rootfs cloning to prevent inode leakage (for hosts with kernel < 5.19 and no shiftfs) (issue #570).
+  * Added --disable-inner-image-preload flag to sysbox-mgr (speeds up Sysbox container startup).
+  * Added --syscont-mode flag to sysbox-mgr; allows Sysbox to work in system container mode (default) or regular container mode; the latter is meant for running microservices with stronger isolation.
+  * Added --disable-shiftfs-on-fuse flag to sysbox-mgr; prevents Sysbox from mounting shiftfs on top of FUSE-backed filesystems (some of which don't work with shiftfs).
+  - Fix emulation of /sys/module/nf_conntrack/parameters inside containers.
```

Comment:
Replace "-" with "*" -- formatting is important in this case for the packaging tool do to its thing.

Reply:
Done.